deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapper.java

```diff
@@ -40,7 +40,8 @@ public class OpenApiClientGeneratorWrapper {
     private String apiPackage = "";
     private String modelPackage = "";
 
-    public OpenApiClientGeneratorWrapper(final Path specFilePath, final Path outputDir, final boolean verbose) {
+    public OpenApiClientGeneratorWrapper(final Path specFilePath, final Path outputDir, final boolean verbose,
```

Comment:
Why change the constructor's interface?

Reply:
I followed the same line as it was done for setting up _verbose_. It can be done via builder too, I don't have a strong preference, either way is fine.